Makefile

```diff
@@ -20,4 +22,7 @@ test_unittest:
 
 test: test_unittest test_integration
 
+coverage_report: test
+	coverage report -m
```

Comment:
Should not we follow `$(PYTHON)` value here like in all other places.

Reply:
I don't see any reasons to add it. Everything works without it.

Reply:
Okay.